<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpidrTpx3Tv</class>
 <widget class="QMainWindow" name="SpidrTpx3Tv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPIDR-TTV</string>
  </property>
  <property name="windowIcon">
   <iconset resource="spidrtpx3tv.qrc">
    <normaloff>:/default/SPIDR-logo.ico</normaloff>:/default/SPIDR-logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="_lbView">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="spidrtpx3tv.qrc">:/default/spidrtpx3tv.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="_actionExit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dwZapper">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>761</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="_pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>SPIDR IP addr</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_leSpidrIpAddr">
       <property name="text">
        <string>192.168.1.10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelDeviceNr">
       <property name="text">
        <string>Device number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="_cbDeviceNr">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Host IP addr</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="_leHostIpAddr">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="_lbPorts">
       <property name="toolTip">
        <string>(Port)</string>
       </property>
       <property name="text">
        <string>Data IP port</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="_lePort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="Line" name="line_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Grayscale Display</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min value (black)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="_sbMinValue">
       <property name="maximum">
        <number>4095</number>
       </property>
       <property name="singleStep">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max value (white)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="_sbMaxValue">
       <property name="maximum">
        <number>16777215</number>
       </property>
       <property name="singleStep">
        <number>64</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="_pbResetImage">
       <property name="text">
        <string>Reset image</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bytes received</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QLabel" name="_lbBytesReceived">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Bytes sampled</string>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="_lbBytesSampled">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Bytes flushed</string>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QLabel" name="_lbBytesFlushed">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Sample size</string>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="QLabel" name="_lbSampleSize">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Packets received</string>
       </property>
      </widget>
     </item>
     <item row="26" column="1">
      <widget class="QLabel" name="_lbPacketsReceived">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Last packet size</string>
       </property>
      </widget>
     </item>
     <item row="27" column="1">
      <widget class="QLabel" name="_lbLastPacketSize">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item row="30" column="0">
      <widget class="QCheckBox" name="_cbTestMode">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Test mode</string>
       </property>
      </widget>
     </item>
     <item row="31" column="0">
      <widget class="QLabel" name="_labelShutter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Shutter</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="31" column="1">
      <widget class="QPushButton" name="_pbShutter">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="33" column="0">
      <widget class="QLabel" name="_labelDacCoarse">
       <property name="text">
        <string>DAC Vthr-coarse</string>
       </property>
      </widget>
     </item>
     <item row="33" column="1">
      <widget class="QSpinBox" name="_sbDacCoarse">
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="36" column="0">
      <widget class="QLabel" name="_labelDacFine">
       <property name="text">
        <string>DAC Vthr-fine</string>
       </property>
      </widget>
     </item>
     <item row="36" column="1">
      <widget class="QSlider" name="_sliderDacFine">
       <property name="maximum">
        <number>511</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="37" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="38" column="0" colspan="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="spidrtpx3tv.qrc">:/default/NIKHEF-logo-transparent.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="39" column="0" colspan="2">
      <widget class="QLabel" name="_lbVersion">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>V0.0.0   01-Nov-2013</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="29" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="_actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="spidrtpx3tv.qrc"/>
 </resources>
 <connections/>
</ui>
